<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mn921channelcfg</class>
 <widget class="QDialog" name="mn921channelcfg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>804</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="mn921">
      <item>
       <layout class="QVBoxLayout" name="ats">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="font">
           <font>
            <pointsize>32</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Настройка ячейки MN921 или  MN825 </string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string>АТС</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="ats_cfg">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Тип АТС набора</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ats_type">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>Тоновый</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Импульсный</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="tch">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ТЧ</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="font">
               <font>
                <pointsize>15</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Тип ТЧ</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="tch_type">
              <property name="font">
               <font>
                <pointsize>15</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>Двухпроводка</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Четырехпроводка-ФЛ</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Четырехпроводка-ТЧ</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="font">
               <font>
                <pointsize>15</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Время вызова</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="tch_timeoc">
              <property name="font">
               <font>
                <pointsize>15</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="font">
               <font>
                <pointsize>15</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Максимальное время вызова</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="tch_maxtimeoc">
              <property name="font">
               <font>
                <pointsize>15</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="font">
               <font>
                <pointsize>15</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Минимальное время вызова</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="tch_mintimeoc">
              <property name="font">
               <font>
                <pointsize>15</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <pointsize>15</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Частота вызова</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="tch_fr">
              <property name="font">
               <font>
                <pointsize>15</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>600</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2100</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="mb">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string>МБ</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="toc_2">
          <item>
           <widget class="QLabel" name="tcfg_2">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Время вызова</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="mb_timeoc">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="cb">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ЦБ</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="toc">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Время вызова</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="cb_timeoc">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout"/>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Р168 радиостанция </string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Время вызова</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="r168_timeoc">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
